operations/iuf/stages/deliver_product.md

```diff
@@ -1,20 +1,13 @@
 # `deliver-product`
 
-The `deliver-product` stage delivers the content provided in the product distribution file to the system. Artifacts will be uploaded to the following locations based on entries specified in the product `iuf-product-manifest.yaml` file.
+The `deliver-product` stage delivers the content provided in the product distribution files to the system. The versioned product catalog entry for each product is updated as the new content is uploaded.
 
-| Location                      | `iuf-product-manifest.yaml` Entry |
-| ----------------------------- | --------------------------------- |
-| S3 Content                    | `s3`                              |
-| S3 Loftsman Manifests         | `loftsman`                        |
-| Nexus Docker Registries       | `docker`                          |
-| Nexus Helm Chart Repositories | `helm`                            |
-| Nexus RPM Blob Stores         | `nexus_blob_stores`               |
-| Nexus RPM Repositories        | `nexus_repositories`              |
-| Nexus RPMs                    | `rpms`                            |
-| VCS Content                   | `vcs`                             |
-| IMS Images and Recipes        | `ims`                             |
+`deliver-product` details are explained in the following sections:
 
-The versioned product catalog entry for the product is updated as the new content is uploaded.
+- [Impact](#impact)
+- [Input](#input)
+- [Execution Details](#execution-details)
```

Comment:
Per HPE style guide, we use sentence casing for section headers (except for the overall page header)

```suggestion
- [Execution details](#execution-details)
```